### ISSUE TITLE
Allow BG SEO to Review Category Pages

### ISSUE THREAD

**Issue by john-paulb** [BoldGrid/boldgrid-seo#22]
It would be beneficial to allow the BG SEO tool to also review category pages for SEO best practices.

**Comment by timelsass** [comment]
I agree, I mentioned this in reference to https://github.com/BoldGrid/boldgrid-seo/issues/21#issuecomment-573487502 as well as many competitor's products offer options to adjust various things on categories and other dynamic pages
